docs/main/troubleshooting/troubleshooting-local-storage.md

```diff
@@ -242,14 +244,22 @@ Set the reboot timer schedule at different time i.e. staggered at various interv
 
 ### How to fetch the OpenEBS Dynamic Local Provisioner logs?
 
-**Workaround:**
+**Workaround**
 
 Review the logs of the OpenEBS Local PV provisioner. OpenEBS Dynamic Local Provisioner logs can be fetched using:
 
 ```
 kubectl logs -n openebs -l openebs.io/component-name=openebs-localpv-provisioner
 ```
 
+### Unable to mount `xfs` File System
```

Comment:
Don't we need similar thing for the replicated storage?
@avishnu @dsharma-dc

Reply:
Fair point, I'd think yes we need. I wonder why it's never reported with Mayastor though(or has it?) while lvm has so many reports.

Reply:
Not sure, maybe users are already be on higher kernel versions?
Also also it's probably "worse" on mayastor as I think the xfs utils may be even newer. Also more problematic there because volumes can move between nodes! :(